invenio_app_rdm/config.py

```diff
@@ -651,8 +656,17 @@ def files_rest_permission_factory(obj, action):
         "content-type": "application/vnd.geojson+json",
         "filename": "{id}.geojson",
     },
+    "marcxml": {
```

Comment:
major: I think something went wrong with the rebasing, if you see the resulting file it has the `marcxml` entry duplicated. There should be only one (the one from the previous PR).

https://github.com/inveniosoftware/invenio-app-rdm/blob/357b0ef0c22d0cdb7b0d6a1a29c9e12794e891f3/invenio_app_rdm/config.py#L643